src/gluonnlp/data/stream.py

```diff
@@ -170,8 +170,8 @@ class SimpleDatasetStream(DatasetStream):
         passed to this class.
     file_pattern: str
         Path to the input text files.
-    file_sampler : str, {'sequential', 'random'}, defaults to 'random'
-        The sampler used to sample a file.
+    file_sampler : str or gluon.data.Sampler, defaults to 'random'
```

Comment:
I wonder if "file" is the right unit of the chunk of data. Each chunk in a dataset stream should be a dataset, so the API design should reflect that.

Reply:
I'm not sure what is the best way to define a "chunk". Currently Corpus/Textline dataset both take "file" as an argument to construct a dataset.